src/pages/Home.tsx

```diff
@@ -1,7 +1,116 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { Link, Outlet, useNavigate } from 'react-router-dom';
+import { observer } from "mobx-react-lite";
+import { rootStore } from "../stores/rootStore";
+import logo from '../assets/images/logo.png'
 
-const Home: React.FC = () => {
-  return <div>Home Page</div>;
+const Home: React.FC = observer(() => {
+  const { authStore } = rootStore;
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    // later to set, if user is already logged in redirect to dashboard. if not go to landing page
+  }, []);
+
+  return (
+    <div>
+      <nav style={styles.navbar}>
+        <img src={logo} style={styles.navbarLogo} />
+        <ul style={styles.navbarMenu}>
+          <li style={styles.navbarMenuItem}>
+            <Link to="/landingpage" style={{ ...styles.navbarMenuItemLink, ...(window.location.pathname === '/landingpage' ? styles.activeLink : {}) }}>Welcome</Link>
+          </li>
+          <li style={styles.navbarMenuItem}>
+            <Link to="/dashboard" style={{ ...styles.navbarMenuItemLink, ...(window.location.pathname === '/dashboard' ? styles.activeLink : {}) }}>Dashboard</Link>
+          </li>
+          <li style={styles.navbarMenuItem}>
+            <Link to="/faq" style={{ ...styles.navbarMenuItemLink, ...(window.location.pathname === '/faq' ? styles.activeLink : {}) }}>FAQ</Link>
+          </li>
+          <li style={styles.navbarMenuItem}>
+            <Link to="/about" style={{ ...styles.navbarMenuItemLink, ...(window.location.pathname === '/about' ? styles.activeLink : {}) }}>About</Link>
+          </li>
+          <li style={styles.navbarMenuItem}>
+            <Link to="/watchdemo" style={{ ...styles.navbarMenuItemLink, ...(window.location.pathname === '/watchdemo' ? styles.activeLink : {}) }}>Watch Demo</Link>
+          </li>
+        </ul>
+        <div style={styles.navbarEnd}>
+          {authStore.user ? (
+            <div>
+              <button style={styles.logOutButton} onClick={authStore.logout}>Logout</button>
+            </div>
+          ) : (
+            <div>
+              <Link to="/login" style={styles.smallerButton}>Login</Link>
+              <Link to="/signup" style={styles.smallerButton}>Sign Up</Link>
+            </div>
+          )}
+        </div>
+      </nav>
+      <Outlet />
+    </div>
+  );
```

Comment:
nav bar or header bar is usually a file on its own, usually in components.

Reply:
Done